Tests/PreTokenizerTests/PreTokenizerTests.swift

```diff
@@ -171,4 +171,24 @@ class PreTokenizerTests: XCTestCase {
             ["▁Hey", "▁my", "▁friend", "▁", "▁<s>", "▁how", "▁are", "▁you"]
         )
     }
+
+    func testBertPreTokenizer() {
+        let preTokenizer1 = BertPreTokenizer(config: Config([:]))
+        XCTAssertEqual(
+            preTokenizer1.preTokenize(text: "Hey friend!"),
+            ["Hey", "friend", "!"]
+        )
+        XCTAssertEqual(
+            preTokenizer1.preTokenize(text: "Hey friend!     How are you?!?"),
+            ["Hey", "friend", "!", "How", "are", "you", "?", "!", "?"]
+        )
+        XCTAssertEqual(
+            preTokenizer1.preTokenize(text: "   Hey,    friend ,    what's up?  "),
+            ["Hey", ",", "friend", ",", "what", "\'", "s", "up", "?"]
+        )
+        XCTAssertEqual(
+            preTokenizer1.preTokenize(text: "   Hey,    friend ,  0 99  what's up?  "),
+            ["Hey", ",", "friend", ",", "0", "99", "what", "\'", "s", "up", "?"]
+        )
+    }
```

Comment:
Original Rust implementation also had this test case scenario:

```
        XCTAssertEqual(
            preTokenizer.preTokenize(text: "野口里佳 Noguchi Rika"),
            ["野", "口", "里", "佳", "Noguchi", "Rika"]
        )
```

Reply:
I noticed that as well, but figured that the architecture here is a bit different and it should be handled by `BertNormalizer`, is this assumption correct @pcuenca?

Reply:
Yes, I think this is great as it is, we can always iterate if needed.